src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -72,6 +74,14 @@
     @Argument(fullName = StandardArgumentDefinitions.DISABLE_SEQUENCE_DICT_VALIDATION_NAME, shortName = StandardArgumentDefinitions.DISABLE_SEQUENCE_DICT_VALIDATION_NAME, doc = "If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!", optional = true, common = true)
     private boolean disableSequenceDictionaryValidation = false;
 
+    @Argument(fullName = StandardArgumentDefinitions.ERROR_ON_OUT_OF_DATE_INDEX,
```

Comment:
I might rename this to `ERROR_ON_OUT_OF_DATE_INDEX_LONGNAME` to match convention and to avoid confusing it with `ERROR_ON_OUT_OF_DATE_INDEX_DEFAULT`

Reply:
Fixed!